agency-api-spec.md

```diff
@@ -331,27 +440,24 @@ $ curl -X POST -H "Content-Type: application/json" -d @- https://foia-api.agency
         }
     ],
     "contract_number": "5547",
```

Comment:
any reason this `contract_number` field and the `region` field below are in the sample request?

Reply:
Hmm, these represent the 10% fields from the GSA example above. I suppose this should be a generic example and we should leave them out.